### PR TITLE
Update multiple_reactions_engagement's alignment

### PR DESCRIPTION
**Pull request by Kireeti-28**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

Resolves https://github.com/forem/forem/issues/19474

## QA Instructions, Screenshots, Recordings

- Issue: On iPhone 8, seems like emojis were not aligned with the numbers next to them and was overlapping on text below them.
_image_

- Resolution: Made use of flex-box to show the emojis + number combination on UI which eliminates unwanted split between emojis followed by its number.
_image _

### UI accessibility concerns?

- Just made use of flex-box which I felt to be an ideal solution for the misalignment issue here.
- From accessibility standpoint, I tested this in smallest screen possible and it worked just fine without any overlapping.
image

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by Ridhwana** [comment]
HI @Kireeti-28 thank you for submitting this PR. I'm looping in our designer @anujbhavsar96 to give this a look.

**Review by anujbhavsar96 — APPROVED** [review]
This looks great @Kireeti-28 🚀 Thank you for this.

**Review by filleduchaos — CHANGES_REQUESTED** [review]
Checked it out locally and while the layout is consistent on desktop and mobile 🎉 However, this change causes reactions with zero counts to show up as well (see before/after comparison below)

Reactions list before
Reactions list after

File: app/assets/stylesheets/views/article.scss [hunk 1]
```diff
@@ -473,18 +473,15 @@
...
-  span {
-    margin-right: 1em;
-    vertical-align: middle;
-  }
-  svg {
-    display: inline-block;
-  }
-  img {
-    vertical-align: middle;
+   & > span {
```
[the remark this answers is not in the document]
Comment (Kireeti-28) [reply app/assets/stylesheets/views/article.scss]: Corrected both the code indentation plus showing up reactions with zero count via commit: https://github.com/forem/forem/pull/19484/commits/8508820c2da627ad398e3911047dc8dd0a2226e6

image


Apologies for not having closer look at first place.

File: app/views/articles/_multiple_engagements.html.erb [hunk 2]
```diff
@@ -6,5 +6,7 @@
     </span>
   <% end %>
 
-  <span class="placeholder" style="width: 24px; height: 24px;">&nbsp;</span>
+  <% if false %>
+  <%# <span class="placeholder" style="width: 24px; height: 24px;">&nbsp;</span> %>
+  <% end %>
```
Comment (filleduchaos) [inline app/views/articles/_multiple_engagements.html.erb:11]: Oops I forgot to mention that you can take these out, since they will never be rendered

**Review by filleduchaos — APPROVED** [review]
🎉 Excellent!

**Pull request merged**